epictrack-api/tests/conftest.py

```diff
@@ -159,14 +158,9 @@ def restart_savepoint(sess2, trans):
         yield sess
 
 
-def mock_decorator(f, *args, **kwargs):
-    """Function to mock a decorator. Used to mock auth.require"""
-
-    @wraps(f)
-    def decorated_function(*args, **kwargs):
-        return f(*args, **kwargs)
-
-    return decorated_function
-
-
-patch("api.utils.auth.require", mock_decorator, spec=True).start()
+@pytest.fixture()
+def staff_admin_headers(client, jwt, request):
```

Comment:
Shall we have a fixture factory created else where and call it in the conftest instead of this. Better we maintain multiple classes for individual headers and stack it in single class and call the configure() method of that class in conftest. In that way we don't want to over crowd the conftest with such specific impelementations